inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java

```diff
@@ -172,6 +175,21 @@ public ActiveLearningSidebar(String aId, IModel<AnnotatorState> aModel,
         super(aId, aModel, aActionHandler, aJCasProvider, aAnnotationPage);
 
         annotationPage = aAnnotationPage;
+        project = aModel.getObject().getProject();
```

Comment:
Do not copy the values (have them as fields in the component) - always retrieve them from the model object when needed. You may keep them for a short time in local variables, but not in class fields.